exercises/en/test_01_09b.py

```diff
@@ -1,14 +1,15 @@
+import re
+
 def test():
     # Here we can either check objects created in the solution code, or the
     # string value of the solution, available as __solution__. A helper for
     # printing formatted messages is available as __msg__. See the testTemplate
     # in the meta.json for details.
     # If an assertion fails, the message will be displayed
 
-    # Since we haven't started assigning charts to variable names yet,
-    # this might be the better way to test for the first exercise.
-    # Maybe even for later exercises.
-
-    # TODO implement test
-    __msg__.good("Not yet implemented")
-    # __msg__.good("You're correct, well done!")
+    # Regex expression to capture the line plot variable name without whitespace
+    capture_grp = re.search("[^\s]*(?=\s*=\s*alt.Chart\(stocks\).mark_line)", __solution__)
```

Comment:
Nice!

Reply:
One small comment, and (@ecool50 can attest to this) is I usually like as mix of both testing if a string is in __solution__ as well as checking the object itself. The last few questions we did not assign to a variable but when we do I like a mix of the two. Just something to keep in mind.